### ISSUE TITLE
Reconsider the use of Angular 2 animations

### ISSUE THREAD

**Issue by matebordas** [rabix/cottontail#73]
We should re-consider the use the Angular 2 animations because it lacks some browser support (Safari doesn't support it): 

http://caniuse.com/#feat=web-animation
https://angular.io/docs/ts/latest/guide/animations.html

**Comment by ivanbatic** [comment]
IMO, animations are at the bottom of our priorities.
Our worries for Q3 and Q4 are 
- Full feature parity with the SBG platform editors
- Distribution for desktop and platform with different environments integration
- Layout design and stabilization
- Hi-res graphical design at the end

Animations are the UI sugar that I frankly don't think that we'll be spending significant efforts on until late Q3 or Q1 2017. Then, we'll see what we want to animate and what's the best way to achieve those effects. For example, our current Web Animation API usages can be easily done with CSS transitions.
Then, if there is something that we want to achieve and the way to do it is the WA API, we can check the cross-browser policies to determine if that's a thing that must work across all browsers.

So basically, you can experiment play with them, but expect it to be disposable code and don't spend much time or get emotionally attached to it.

**Comment by matebordas** [comment]
In that case I will just close this issue.
